packages/@aws-cdk/aws-iot-actions/test/s3/s3-action.test.ts

```diff
@@ -0,0 +1,213 @@
+import { Template } from '@aws-cdk/assertions';
+import * as iam from '@aws-cdk/aws-iam';
+import * as iot from '@aws-cdk/aws-iot';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as cdk from '@aws-cdk/core';
+import * as actions from '../../lib';
+
+test('Default s3 action', () => {
+  // GIVEN
+  const stack = new cdk.Stack();
+  const topicRule = new iot.TopicRule(stack, 'MyTopicRule', {
+    sql: iot.IotSql.fromStringAsVer20160323("SELECT topic(2) as device_id FROM 'device/+/data'"),
+  });
+  const bucket = new s3.Bucket(stack, 'MyBucket');
```

Comment:
Again, I would suggest using an imported Bucket for these tests, the assertions in the test will be easier to write that way.